Sources/Swift/SwiftDescriptor.swift

```diff
@@ -8,4 +8,20 @@ public class SwiftDescriptor: NSObject {
         return String(describing: type(of: object))
     }
     
+    @objc
+    public static func getSwiftErrorDescription(_ error: Error) -> String? {
+        let description = String(describing: error)
+        
+        // We can't reliably detect what is PII in a struct and what is not.
+        // Furthermore, we can't detect which property contains the error enum.
+        if description.contains(":") || description.contains(",") {
+            return nil
+        }
```

Comment:
h: This would also return nil for Enum errors with arguments making the stripping below unnecessary.

Reply:
I probably need a test with an enum like this

```Swift
enum LoginError: Error {
    case wrongUser(id : String)
    case wrongPassword
}
```

Reply:
Thanks will add one next week.

Reply:
I ditched the whole logic so this is comment is unrelevant.